### ISSUE TITLE
PSP process

### ISSUE THREAD

**Issue by Noc2** [maciejhirsz/uos#6]
Could we potentially link/integrate this into the [PSP process](https://github.com/w3f/PSPs)? It’s hard to find here. (or is already shared in other ways?)

**Comment by maciejhirsz** [comment]
@Noc2 absolutely. I think it makes sense to have a separate repo for Ethereum support (though that repo doesn't have to sit in my username necessarily), but the Polkadot/Substrate part of things can be either integrated into PSP, or defined in that process and only linked over here.
